solutions/src/main/java/ru/sberbank/school/task02/FxConverterFactory.java

```diff
@@ -0,0 +1,19 @@
+package ru.sberbank.school.task02;
+
+import java.math.BigDecimal;
+
+import ru.sberbank.school.task02.util.ClientOperation;
+import ru.sberbank.school.task02.util.ExternalQuotesServiceDemo;
+import ru.sberbank.school.task02.util.Symbol;
+
+public class FxConverterFactory implements ServiceFactory {
+  @Override
+  public FxConversionService getFxConversionService(ExternalQuotesService externalQuotesService) {
+    return new FxConverter(externalQuotesService);
+  }
+
+  public static void main(String[] args) {
```

Comment:
&#39;method def modifier&#39; has incorrect indentation level 2, expected level should be 4.

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a-1737658297>*